Ginger/GingerCoreNET/Application Models/Learn/POM/PomLearnUtils.cs

```diff
@@ -272,6 +272,11 @@ public async Task Learn()
             PrepareLearningConfigurations();
             LearnScreenShot();
             POM.PageURL = ((AgentOperations)Agent.AgentOperations).Driver.GetURL();
+            string normalizedPageUrl = Path.GetFullPath(new Uri(POM.PageURL).LocalPath).TrimEnd(Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar);
```

Comment:
Make sure this code executes in case of local path only

Reply:
done